docs/intro.rst

```diff
@@ -59,63 +59,64 @@ quantum processing unit (QPU).
 >>> from dwave.system import DWaveSampler
 >>> import dwave.inspector
 ...
->>> # Get solver
->>> sampler = DWaveSampler(solver=dict(qpu=True))
+>>> # Get solver 
```

Comment:
trailing space

Reply:
Removed in [this commit](https://github.com/dwavesystems/dwave-inspector/pull/155/commits/da613974be0270874fd5f821c62ecf255ad23f0b)